### PR TITLE
Enable FindBugs in the local build flows.

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Just a POM patch, I decided not to integrate fixes now and just leave it to you.

@KostyaSha @lanwen

### CODE REVIEW

**Comment by lanwen** [comment]
ok

File: pom.xml [hunk 1]
```diff
@@ -208,6 +211,25 @@
...
                 </configuration>
             </plugin>
+            
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>findbugs-maven-plugin</artifactId>
+                <version>${findbugs-maven-plugin.version}</version>
+                <configuration>
+                    <xmlOutput>true</xmlOutput>
+                    <failOnError>${findbugs.failOnError}</failOnError>
```
Comment (KostyaSha) [inline pom.xml]: Why do you need redefine variable if you already set in property?

Comment (oleg-nenashev) [inline pom.xml]: I just keep it explicit, so it improves the POM file readability

Comment (lanwen) [inline pom.xml]: i've think it is to have ability to redefine with help of `-Dfindbugs.failOnError` ... if it works without this property - it should be inlined, as it doesn't encrease readability

Comment (oleg-nenashev) [inline pom.xml]: -Dfindbugs.failOnError can be redefined from the external run with such approach even if you don't specify this line explicitly

If you disagree with this string, I'll remove it

Comment (lanwen) [inline pom.xml]: for what we may need to use it in external build?

File: pom.xml [hunk 2]
```diff
@@ -208,6 +211,25 @@
...
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>run-findbugs</id>
```
Comment (KostyaSha) [inline pom.xml]: And why do you need define it if it
```
findbugs:check
Full name:

org.codehaus.mojo:findbugs-maven-plugin:3.0.2:check

Description:

(no description)
Attributes:
Binds by default to the lifecycle phase: verify.
```

File: pom.xml (hunk 1 above)
Comment (lanwen) [inline pom.xml]: @oleg-nenashev (it may be used as separate goal if we wanna external use)

Comment (KostyaSha) [inline pom.xml]: How he didn't wrap it into profile yet...

**Pull request closed without merging**